src/main/webapp/script.js

```diff
@@ -658,3 +685,9 @@ async function displayPoliticianPage(imgUrl) {
 
     returnToFeed(repName, document.getElementById("feed"));
 }
+
+//Logs out a user or rep
+function resetLocalStorage() {
```

Comment:
Some changes here are also in another PR. Remember to Keep your workspace and PRs focused. If PRs are not getting approved fast enough, then let us know.